examples/dynamics/throwing_while_standing_on_the_ice.py

```diff
@@ -0,0 +1,77 @@
+from sympy import solve, Symbol, Eq
+from symplyphysics import print_expression
+from symplyphysics.laws.dynamics import friction_force_from_normal_force as friction_force
+from symplyphysics.laws.dynamics import mechanical_work_from_force_and_move as work_friction
+from symplyphysics.laws.dynamics import kinetic_energy_from_mass_and_velocity as kinetic_energy
+from symplyphysics.definitions import momentum_is_mass_times_velocity as momentum
+from symplyphysics.laws.conservation import momentum_after_collision_equals_to_momentum_before as momentum_conservation
+from symplyphysics.laws.conservation import mechanical_energy_after_equals_to_mechanical_energy_before as energy_conservation
+from symplyphysics.laws.dynamics import force_reaction_from_force_action as third_newton_law
+
+# Example from https://uchitel.pro/%D0%B7%D0%B0%D0%B4%D0%B0%D1%87%D0%B8-%D0%BD%D0%B0-%D0%B7%D0%B0%D0%BA%D0%BE%D0%BD-%D1%81%D0%BE%D1%85%D1%80%D0%B0%D0%BD%D0%B5%D0%BD%D0%B8%D1%8F-%D0%B8%D0%BC%D0%BF%D1%83%D0%BB%D1%8C%D1%81%D0%B0/
+# A skater with a mass of M = 70 kg, standing on the ice,
+# throws a puck with a mass of m = 0.3 kg horizontally at a speed of v = 40 m/s.
+# How far s will the skater roll back if the coefficient of friction of the skates
+# on the ice is μ = 0.02?
+
+mass_of_skater = Symbol("mass_of_skater")
+mass_of_puck = Symbol("mass_of_puck")
+friction_factor = Symbol("friction_factor")
+velocity_of_puck = Symbol("velocity_of_puck")
+distance = Symbol("distance")
+
+gravity_acceleration = Symbol("gravity_acceleration")
+
+velocity_of_skater = Symbol("velocity_of_scater")
+
+momentum_of_skater = momentum.definition.subs({
+    momentum.mass: mass_of_skater,
+    momentum.velocity: velocity_of_skater
+}).rhs
+momentum_of_puck = momentum.definition.subs({
+    momentum.mass: mass_of_puck,
+    momentum.velocity: velocity_of_puck
+}).rhs
+
+momentum_conservation_law = momentum_conservation.law.subs({
+    momentum_conservation.momentum(momentum_conservation.time_before): momentum_of_skater,
+    momentum_conservation.momentum(momentum_conservation.time_after): momentum_of_puck
+})
+
+
+velocity_of_skater_law = solve(momentum_conservation_law, velocity_of_skater, dict=True)[0][velocity_of_skater]
+
+reaction_force_value = third_newton_law.law.subs({
+    third_newton_law.force_action: -mass_of_skater * gravity_acceleration
```

Comment:
Explain minus, or use absolute value of reaction force.

Reply:
In the code of the module with this law, it is not indicated that the values of Newton's 3rd law are vectors. There, the scalar reaction force is equal to the scalar action force multiplied by -1. Although the minus sign in the formula is appropriate for the vector form of Newton's 3rd law. The minus sign says that the vectors in question have opposite directions. But these forces are equal in modulus. Because of this moment, I had to add a "-".

Reply:
Yes, but as soon as we have a modified version of 3rd law, where minus sign represents direction, we should apply modulus (take absolute value) to introduce result to friction force law.

Reply:
Use `abs()` function. And add comment to the code, why we are doing so.

Reply:
Thanks for the tip!

Reply:
> Используйте `abs()`функцию. И добавим комментарий к коду, почему мы так делаем.

Like this?
```python
reaction_force_value = third_newton_law.law.subs({
    third_newton_law.force_action: mass_of_skater * gravity_acceleration
}).rhs

# We take only the modulus of the reaction force vector, 
# since we set the direction of the vertical axis coinciding with the direction of this vector
friction_force_value = friction_force.law.subs({
    friction_force.friction_factor: friction_factor,
    friction_force.normal_reaction: abs(reaction_force_value)
}).rhs
```

Reply:
Yes